packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -315,6 +315,8 @@ function FontCollection( { slug } ) {
 								</FlexItem>
 								<FlexItem>
 									<SelectControl
+										__nextHasNoMarginBottom
+										__next40pxDefaultSize
```

Comment:
## Testing instructions

In Global Styles, go to Typography ▸ Manage Fonts ▸ Install Fonts.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/ed63d9a8-2e9c-4eb1-8afa-6a5ae8290402" alt="Install Fonts, before" width="639">|<img src="https://github.com/user-attachments/assets/a8046ef3-9da9-4eed-9d05-e7afb168d11b" alt="Install Fonts, after" width="640">|